### PR TITLE
Init benchmarks module with multi-projects structure

### LINKED ISSUE #135: Refactor build.sbt
At the moment `build.sbt` is a little mess. If we want to extend it with additional plugins and functionality, it could be cool to have a more structured build definition and explicit build order.

- Rewrite to `lazy val root = project...`
- Separate root / website and maybe tests / linting into `lazy val root / docs = `;
- Separate Laika configuration from `build.sbt` to another place;

### PR DESCRIPTION
**Pull request by zeotuan**
Init benchmark project as part of #130

Close #135

### CODE REVIEW

File: benchmarks/src/main/scala/com/github/mrpowers/spark/fast/tests/MyBenchmark.scala [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+private class MyBenchmark extends DataFrameComparer {
+  @Benchmark
+  @BenchmarkMode(Array(Mode.AverageTime, Mode.SingleShotTime))
+  @Fork(value = 2)
+  @Warmup(iterations = 10)
+  @Measurement(iterations = 10)
+  @OutputTimeUnit(TimeUnit.NANOSECONDS)
+  def assertApproximateDataFrameEqualityWithPrecision(blackHole: Blackhole): Boolean = {
+    val spark = SparkSession
```
Comment (zeotuan) [inline benchmarks/src/main/scala/com/github/mrpowers/spark/fast/tests/MyBenchmark.scala:18]: Should we also import core project test dependency to reuse some of the already defined test utility?

**Comment by alfonsorr** [comment]
I really like the idea of having a multi module system. @SemyonSinchenko , what do you think about this approach?

**Comment by SemyonSinchenko** [comment]
> I really like the idea of having a multi module system. @SemyonSinchenko , what do you think about this approach?

That was my plan: https://github.com/mrpowers-io/spark-fast-tests/issues/135
I like it too

**Review by SemyonSinchenko — APPROVED** [review]
LGTM, thank you!

**Review by alfonsorr — APPROVED** [review]

**Pull request merged**